segment.py

```diff
@@ -0,0 +1,115 @@
+from pyannote.audio import Audio
+from pyannote.core import Segment
+import torch, torchaudio
+import torch.nn.functional as F
+import ffmpeg
+import os
+import time, subprocess
+import glob
+import soundfile as sf
+
+#pyannote Audio class (pyannote/audio/core/io.py)
+
+#mono parameter: in case of multi-channel audio, turn into mono audio by randmoly choosing with 'random' or averaging 
+#all channels with 'downmix' 
+audio = Audio(mono='downmix') #add parsing arg (for nemo we need mono)
+
+def seconds_to_hms(seconds):
+  '''
+  Given an int/float number of seconds it returns a string with the format HH-MM-SS.
+  '''
+
+  filename_format = time.strftime('%H-%M-%S', time.gmtime(seconds))
+
+  return filename_format
+
+def segment_audio(trans_segment,
```

Comment:
Are you creating both wav and pt here? I can only trace back the .wav bit - but for that, you shouldn't need torchaudio, do you?